.pre-commit-config.yaml

```diff
@@ -0,0 +1,20 @@
+---
+repos:
+  - repo: local
+    hooks:
+      - id: "dotnet-format"
+        name: "Format Tests"
+        language: "system"
+        entry: "dotnet format tests/Authzed.Net.Tests --include"
+        types_or: ["c#", "vb"]
+        files: "tests"
+      - id: "dotnet-format"
+        name: "Format Examples"
+        language: "system"
+        entry: "dotnet format examples/protected-api/Api --include"
+        types_or: ["c#", "vb"]
+        files: "examples"
```

Comment:
This uses the locally-installed dotnet rather than having pre-commit pull in the formatter.